Sources/FetchNodeDetails/Contract/TorusLookup.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import web3
+import BigInt
+import PromiseKit
+
+public enum TorusLookupContract{
+    public struct getNodeSet:ABIFunction{
+        public static var name: String = "getNodeSet"
+        public var gasPrice: BigUInt?
+        public var gasLimit: BigUInt?
+        public var contract: EthereumAddress
+        public var from: EthereumAddress?
+        public var verifier:String
+        public var hashedVerifierId:Data
```

Comment:
Add linting everywhere and follow consistent pattern

Reply:
Done